packages/hiro-graph-client/index.d.ts

```diff
@@ -448,6 +448,97 @@ export interface VariablesServlet {
   define<T = any>(options: DefineVariableOptions): Promise<T>;
 }
 
+export type ActionLogType =
+  | 'SearchKI'
+  | 'SearchTask'
+  | 'SearchTeachingSession'
+  | 'DeployKI'
+  | 'UndeployKI'
+  | 'WriteKI'
+  | 'ForkKI'
+  | 'OpenTeachingSession'
+  | 'OwnTeachingSession'
+  | 'StartTeachingSession'
+  | 'ContinueTeachingSession'
+  | 'StartConversionSession'
+  | 'ContinueConversionSession'
+  | 'AssignKI';
+
+export interface BaseEvent<T extends ActionLogMeta> {
+  action: ActionLogType;
+  instanceId: string;
```

Comment:
should it be `instance_id`? as Vitaly mentioned

Reply:
no, I've checked, it should be `instanceId`